cmd/hack/config/cfg.go

```diff
@@ -1,13 +1,11 @@
-package main
+package config
```

Comment:
BTW, `cfg` here stands for `Control Flow Graph` and not for `config` :)

Reply:
can rename any way. the refactoring was done to be able to run hack as a single command `go run ...`